FoodPin/FoodPin/Controller/RestaurantDetailViewController.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  RestaurantDetailViewController.swift
+//  FoodPin
+//
+//  Created by Andreea Sburlea on 27.04.2022.
+//
+
+import UIKit
+
+class RestaurantDetailViewController: UIViewController {
+
+    @IBOutlet private var restaurantImageView: UIImageView!
+    @IBOutlet private var restaurantName: UILabel!
+    @IBOutlet private var restaurantType: UILabel!
+    @IBOutlet private var restaurantLocation: UILabel!
+    var restaurantImageName = ""
```

Comment:
As the hint suggest, try to pass the data using a restaurant object instead of 4 separate properties